third-party/Makefile

```diff
@@ -3,6 +3,9 @@ export CHPL_MAKE_HOME=$(shell pwd)/..
 endif
 include $(CHPL_MAKE_HOME)/make/Makefile.base
 
+# do not include chpl-venv/Makefile.include from Makefile.base
```

Comment:
I'd remove this comment, and put the include directly under the Makefile.base include